src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -164,6 +159,36 @@ ITK_coordinates_to_STIR_physical_coordinates
   return stir_coord;
 }
 
+/* Convert an ITK Pixel (i.e., float) to a STIR Pixel. */
+typename STIRImageSingle::pixel_type
+ITK_pixel_to_STIR_pixel(typename ITKImageSingle::PixelType itk_pixel,
+                        const STIRImageSingle &stir_image,
+                        bool is_displacement_field=false)
+{
+  return static_cast<typename STIRImageSingle::pixel_type>(itk_pixel);
+}
+
+/* Specialisation if the pixel is a vector and we want a multi-image */
+typename STIRImageMulti::pixel_type
+ITK_pixel_to_STIR_pixel(typename ITKImageMulti::PixelType itk_pixel,
+                        const STIRImageMulti &stir_image,
+                        bool is_displacement_field=false)
+{
+  // ITK VariableLengthVector to ITK FixedArray
+  // We know it is length 3
```

Comment:
could add `assert`

Reply:
Done, but I don't actually have any multi images to test with. @rijobro could you possible run this over some data again sorry?

Reply:
Done, @rijobro could you possibly run this on a deformation field again to check? Sorry